src/calculateRentalCost.js

```diff
@@ -4,7 +4,30 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+  const costCar = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days < 3) {
+    sum = costCar * days;
+
+    return sum;
+  }
+
+  if (days >= 3 && days <= 6) {
+    sum = costCar * days - threeDaysDiscount;
+
+    return sum;
+  }
+
+  if (days >= 7) {
+    sum = costCar * days - sevenDaysDiscount;
+
+    return sum;
+  }
+
+  return sum;
```

Comment:
The final `return sum;` is unnecessary because all previous conditions return early, and the function will never reach this point. You can safely remove this line.